internal/hcp/registry/types.builds.go

```diff
@@ -64,6 +65,7 @@ func (b *Build) MergeLabels(data map[string]string) {
 		//if _, ok := build.Labels[k]; ok {
 		//continue
 		//}
+		log.Printf("adding label %q=%q", k, v)
```

Comment:
is this a debugging log?

Reply:
It is, I forgot to remove it, good catch!